code-studio/src/js/components/progress/stage_progress.jsx

```diff
@@ -0,0 +1,59 @@
+/* global React, dashboard */
+
+window.dashboard = window.dashboard || {};
+
+window.dashboard.STAGE_PROGRESS_TYPE = React.PropTypes.arrayOf(React.PropTypes.shape({
+  title: React.PropTypes.oneOfType([React.PropTypes.string, React.PropTypes.number]),
+  status: React.PropTypes.string,
+  kind: React.PropTypes.oneOf(['unplugged', 'assessment', 'puzzle']),
+  link: React.PropTypes.string,
+  id: React.PropTypes.number
+}));
```

Comment:
Fixed in https://github.com/code-dot-org/code-dot-org/commit/7fa5bf6572e349d536a90528db82877ac286944a.